Source/Mosa.Utility.Configuration/CommandLineArguments.cs

```diff
@@ -165,10 +167,11 @@ private static List<Argument> GetMap()
 				new Argument() { Name = "-syslinux", Setting = "Image.BootLoader", Value = "syslinux3.72"},
 				new Argument() { Name = "-syslinux3.72", Setting = "Image.BootLoader", Value = "syslinux3.72"},
 				new Argument() { Name = "-syslinux6.0", Setting = "Image.BootLoader", Value = "syslinux6.03"},
+				new Argument() { Name = "-limine", Setting = "Image.BootLoader", Value = "limine"},
 
 				// Advance:
 				new Argument() { Name = "-hunt-corlib", Setting = "Launcher.HuntForCorLib", Value = "true"},
-				new Argument() { Name = "-plug-korlib", Setting = "Advanced.PlugKorlib", Value = "true"},
+				new Argument() { Name = "-plug-korlib", Setting = "Launcher.PlugKorlib", Value = "true"},
```

Comment:
is this change correct? I don't see another change for changing the setting name

Reply:
Yes, if you make a global file search for Advanced.PlugKorlib, you would've only seen the CLI argument in that class. But if you do for Launcher.PlugKorlib, you'll see a lot more matches. I think this option was simply forgotten to be renamed.